core/domain/exp_services.py

```diff
@@ -705,6 +705,14 @@ def apply_change_list(
                         raise Exception(
                             'Expected recorded_voiceovers to be a dict, '
                             'received %s' % change.new_value)
+                    if (
```

Comment:
Why not `if not is_voiceover_allowed_for_exploration(exploration_id)`?

Reply:
Done